server/model/traces.go

```diff
@@ -25,7 +25,7 @@ func NewTrace(traceID string, spans []Span) Trace {
 
 	rootSpans := make([]*Span, 0)
 	for _, span := range spanMap {
-		parentID := span.Attributes["parent_id"]
+		parentID := span.Attributes[string(TracetestMetadataFieldParentID)]
```

Comment:
this values are not the same. the original key is `parent_id`, but the value defined in the constant is `tracetest.span.parent_uid`. Is this intentional? Doesn't this changes the key?

Reply:
Fixed!